pkg/entity/data_connector/s3Bucket.go

```diff
@@ -0,0 +1,46 @@
+package data_connector
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/strmprivacy/api-definitions-go/v2/api/entities/v1"
+	"strmprivacy/strm/pkg/util"
+)
+
+const assumeRoleArnFlag = "assume-role-arn"
+
+func createS3BucketCmd() *cobra.Command {
+	s3Bucket := &cobra.Command{
+		Use:               "s3 [data-connector-name] [bucket-name]",
+		Short:             "Create a Data Connector for an AWS S3 Bucket",
+		Long:              longDoc,
+		DisableAutoGenTag: true,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			printer = configurePrinterForType(cmd, cmd.Parent().Parent().Name())
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			credentials := readCredentialsFile(cmd.Flags())
+			dataConnectorName := &args[0]
+			bucketName := &args[1]
+
+			dataConnector := &entities.DataConnector{
+				Ref: ref(dataConnectorName),
+				Location: &entities.DataConnector_S3Bucket{
+					S3Bucket: &entities.AwsS3BucketLocation{
+						BucketName:    *bucketName,
+						Credentials:   credentials,
+						AssumeRoleArn: util.GetStringAndErr(cmd.Flags(), assumeRoleArnFlag),
+					},
+				},
+			}
+			create(dataConnector)
+		},
+		Args: cobra.ExactArgs(2),
+	}
+
+	flags := s3Bucket.Flags()
+	flags.String(credentialsFileFlag, "", "file with bucket credentials json")
```

Comment:
Maybe something more descriptive to the specification of this format? IIRC it's a format defined by AWS themselves.